modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -318,14 +322,84 @@ else if ((nestedFieldsDepth > 0) &&
 			}
 		}
 
+		if (nestedFieldsDepth > 0) {
+			List<ObjectRelationship> objectRelationships =
+				_objectRelationshipLocalService.getObjectRelationships(
+					objectDefinition.getObjectDefinitionId());
+
+			Stream<ObjectRelationship> objectRelationshipsStream =
+				objectRelationships.stream();
+
+			List<ObjectRelationship> manyToManyRelationships =
+				objectRelationshipsStream.filter(
+					objectRelationship -> Objects.equals(
+						objectRelationship.getType(),
+						ObjectRelationshipConstants.TYPE_MANY_TO_MANY)
+				).collect(
+					Collectors.toList()
+				);
+
+			manyToManyRelationships.forEach(
+				objectRelationship -> {
+					try {
+						boolean reverse = objectRelationship.isReverse();
+
+						if (objectRelationship.isReverse()) {
+							objectRelationship =
+								_objectRelationshipLocalService.
+									fetchReverseObjectRelationship(
+										objectRelationship, false);
+						}
+
+						Pagination pagination = Pagination.of(1, 20);
+
+						List<com.liferay.object.model.ObjectEntry>
+							manyToManyRelatedObjectEntries =
+								_objectEntryLocalService.
+									getManyToManyRelatedObjectEntries(
+										objectEntry.getGroupId(),
+										objectRelationship.
+											getObjectRelationshipId(),
+										objectEntry.getObjectEntryId(), reverse,
+										pagination.getStartPosition(),
+										pagination.getEndPosition());
+
+						Stream<com.liferay.object.model.ObjectEntry>
+							manyToManyRelatedObjectEntriesStream =
+								manyToManyRelatedObjectEntries.stream();
+
+						map.put(
+							objectRelationship.getName(),
+							manyToManyRelatedObjectEntriesStream.map(
+								objectEntry1 -> {
+									try {
+										return _toDTO(
+											_getDTOConverterContext(
+												dtoConverterContext,
+												objectEntry1.
+													getObjectEntryId()),
+											nestedFieldsDepth - 1,
+											objectEntry1);
+									}
+									catch (Exception exception) {
+										exception.printStackTrace();
```

Comment:
Question: Do we print exceptions like this?

Reply:
Nop. I think you are reviewing the code commit by commit. Careful with that because it is possible that some file changes through the commits. This is an example, the `printStackTrace` was removed and replaced by a login [other commit](https://github.com/liferay-frontend/liferay-portal/pull/2185/commits/f0f29b8e3a81a7c2e509a1f6eb1110ffa34ddb8e) 🙂